cmd/axelard/cmd/vald/tss/tss.go

```diff
@@ -261,13 +264,21 @@ func (mgr *Mgr) ProcessHeartBeatEvent(e tmEvents.Event) error {
 		case exported.Threshold:
 			request = &tofnd.KeyPresenceRequest{
 				KeyUid: string(keyInfo.KeyID),
+				PubKey: []byte{},
 			}
 			response, err = mgr.client.KeyPresence(grpcCtx, request)
 		case exported.Multisig:
-			request = &tofnd.KeyPresenceRequest{
-				KeyUid: fmt.Sprintf("%s_%d", string(keyInfo.KeyID), 0),
+			pubKeys, found := mgr.Keys[string(keyInfo.KeyID)]
+
+			if found {
+				request = &tofnd.KeyPresenceRequest{
+					KeyUid: fmt.Sprintf("%s_%d", string(keyInfo.KeyID), 0),
+					PubKey: pubKeys[0],
+				}
+				response, err = mgr.multiSigClient.KeyPresence(grpcCtx, request)
+			} else {
+				response, err = &tofnd.KeyPresenceResponse{Response: tofnd.RESPONSE_ABSENT}, nil
```

Comment:
```suggestion
			if !found {
			        continue
			}

			request = &tofnd.KeyPresenceRequest{
				KeyUid: fmt.Sprintf("%s_%d", string(keyInfo.KeyID), 0),
				PubKey: pubKeys[0],
			}
			response, err = mgr.multiSigClient.KeyPresence(grpcCtx, request)
```
hacking a response as it is right now is confusing.